CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 
 - Add `loading`, `aria-*` DOM element attributes in `JsxDOM.domProps`: `ariaCurrent`, `ariaInvalid`, `ariaAutocomplete`, etc.
 - Change the internal representation of props for the lowercase components to record. https://github.com/rescript-lang/syntax/pull/665
+- Change the payload of Pconst_char for type safety. https://github.com/rescript-lang/syntax/pull/709
```

Comment:
This PR should be rebases on the latest version of branch `10.1_release` then moved to the section `# 10.1.0-rc.3`.